tests/instrumentation_test.py

```diff
@@ -32,6 +32,7 @@ def onetest(instrumentation: dace.InstrumentationType, size=128):
     C = np.zeros([size, size], dtype=np.float64)
 
     sdfg: dace.SDFG = slowmm.to_sdfg()
+    sdfg._name = f"instrumentation_test_{instrumentation.name}"
```

Comment:
```suggestion
    sdfg.name = f"instrumentation_test_{instrumentation.name}"
```

This should work too I think.

Reply:
Done in 2ec3478f917352bdf1816155caf1da3262b6b0c8